psi-ms.obo

```diff
@@ -23617,6 +23617,208 @@ relationship: has_metric_category MS:4000008 ! ID based metric
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000017 ! chromatogram metric
+
+[Term]
+id: MS:4000154
+name: peptide identification rate of the interquartile RT period
+def: "The identification rate of peptides for the interquartile retention time period, in peptides per second. Higher rates indicate efficient sampling and identification." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000017 ! chromatogram metric
+
+[Term]
+id: MS:4000155
+name: area under TIC
+def: "The area under the total ion chromatogram." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:40000MS ! ID free
+is_a: MS:4000017 ! chromatogram metric
+
+[Term]
+id: MS:4000156
+name: area under TIC RT quantiles
+def: "The area under the total ion chromatogram of the retention time quantiles. Number of quantiles are given by the n-tuple." [PSI:MS]
+is_a: QC:4000004 ! n-tuple
+is_a: MS:4000009 ! ID free
+is_a: MS:4000017 ! chromatogram metric
+
+[Term]
+id: MS:4000157
+name: extent of identified precursor intensity
+def: "Ratio of 95th over 5th percentile of precursor intensity for identified peptides. Can be used to approximate the dynamic range of signal." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: MS:4000158
+name: median of TIC values in the RT range in which the middle half of peptides are identified
+def: "Median of TIC values in the RT range in which half of peptides are identified (RT values of Q1 to Q3 of identifications)" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000001 ! QC metric
+
+
+[Term]
+id: MS:4000159
+name: median of TIC values in the shortest RT range in which half of the peptides are identified
+def: "Median of TIC values in the shortest RT range in which half of the peptides are identified"  [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: QC:4000001 ! QC metric
+
+[Term]
+id: MS:4000160
+name: precursor intensity range
+def: "Minimum and maximum precursor intensity recorded." [PSI:MS]
+is_a: MS:4000009 ! ID free
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+
+[Term]
+id: MS:4000161
+name: identified precursor intensity distribution Q1, Q2, Q3
+def: "From the distribution of identified precursor intensities, the quartiles Q1, Q2, Q3" [PSI:MS]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_value_concept STATO:0000291 ! quantile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000162
+name: unidentified precursor intensity distribution Q1, Q2, Q3
+def: "From the distribution of unidentified precursor intensities, the quartiles Q1, Q2, Q3" [PSI:MS]
+relationship: has_metric_category MS:4000009 ! ID free metric
```

Comment:
`"QC metric not based on identification results."`  -- this does not apply here IMHO.
You need IDs in order to determine what has not been identified.